libkbfs/test_stallers.go

```diff
@@ -38,6 +39,166 @@ const (
 	StallableMDPutUnmerged           StallableMDOp = "PutUnmerged"
 )
 
+type naïveStallContext struct {
```

Comment:
Sounds good; although Op is a bit colliding with MDOps and BlockOps. How about just `naïveStall`?